test/e2e/e2e.go

```diff
@@ -32,11 +32,16 @@ import (
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/v1"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5"
```

Comment:
That's from one of the several rebases that I had over the life of this PR so far.